detection-rules/body_business_email_compromise_new_sender.yml

```
@@ -11,15 +11,26 @@ source: |
   // negating legit replies
   and not (
     (
-      strings.istarts_with(subject.subject, "RE:")
-      // out of office auto-reply
-      // the NLU model will handle these better natively soon
-      or strings.istarts_with(subject.subject, "Automatic reply:")
-      or regex.imatch(subject.subject, '(\[[^\]]+\]\s?){0,3}(re|fwd?|automat.*)\s?:.*')
+      (
+        strings.istarts_with(subject.subject, "RE:")
+        // out of office auto-reply
+        // the NLU model will handle these better natively soon
+        or strings.istarts_with(subject.subject, "Automatic reply:")
+        or regex.imatch(subject.subject, '(\[[^\]]+\]\s?){0,3}(re|fwd?|automat.*)\s?:.*')
+      )
+      and (
+        length(headers.references) > 0
+        or any(headers.hops, any(.fields, strings.ilike(.name, "In-Reply-To")))
```

I know this isn't part of this PR, but just caught my eye
```suggestion
        or headers.in_reply_to is not null
```